src/common/utils.js

```diff
@@ -19,3 +19,33 @@ export const generateValueAdjustments = (priceCrashes = {}, priceSurges = {}) =>
 
     return acc
   }, {})
+
+/**
+ * @param {Array} an array of objects each containing a `weight` property
+ * @returns {Object} one of the items from weightedOptions
+ */
+export function randomChoice(weightedOptions) {
```

Comment:
Can this function be moved to `src/utils.js`? This file is specifically mean for utils that are shared between client and API code due to a quirk of how Vercel builds endpoints. See: https://github.com/jeremyckahn/farmhand/blob/f0c9fb8d5f5fdd33289b3df864670bc8848a2059/src/common/README.md#L1

Right now this is only being used by client code, so it should live with the rest of the client-specific utils.

Reply:
yup, that makes sense! thanks for explaining the difference